app/.vscode/launch.json

```diff
@@ -5,88 +5,88 @@
-      "runtimeArgs": ["run", "--workspace=enso-dashboard", "dev"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-dashboard", "dev"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "GUI",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-gui2", "dev"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["dev:gui"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "GUI (Storybook)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-gui2", "story:dev"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-gui2", "story:dev"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "Dashboard (Build)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-gui2", "build:cloud"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["build:gui"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "Dashboard (E2E UI)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-dashboard", "test:e2e:debug"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-dashboard", "test:e2e:debug"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "GUI (E2E UI)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-gui2", "test:e2e", "--", "--ui"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-gui2", "test:e2e", "--", "--ui"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "Dashboard (All tests)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-dashboard", "test"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-dashboard", "test"]
     },
     {
       "type": "node",
       "request": "launch",
       "name": "Dashboard (E2E tests)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-dashboard", "test:e2e"],
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-dashboard", "test:e2e"],
       "outputCapture": "std"
     },
     {
       "type": "node",
       "request": "launch",
       "name": "Dashboard (Unit tests)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-dashboard", "test:unit"],
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-dashboard", "test:unit"],
       "outputCapture": "std"
     },
     {
       "type": "node",
       "request": "launch",
       "name": "GUI (All tests)",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run", "--workspace=enso-gui2", "test"]
+      "runtimeExecutable": "pnpm",
+      "runtimeArgs": ["run", "--filter", "enso-gui2", "test"]
```

Comment:
I remember @Frizi also wrote about `-r` option (so we recursively search for package) - is it not needed?

Reply:
i have no idea tbh. probably better to just wait for @Frizi to provide insights

Reply:
I just checked and indeed, without `-r` it works.